django/core/management/commands/loaddata.py

```diff
@@ -136,83 +149,79 @@ def loaddata(self, fixture_labels):
         # If we found even one object in a fixture, we need to reset the
         # database sequences.
         if self.loaded_object_count > 0:
-            sequence_sql = connection.ops.sequence_reset_sql(no_style(), self.models)
-            if sequence_sql:
-                if self.verbosity >= 2:
-                    self.stdout.write('Resetting sequences')
-                with connection.cursor() as cursor:
-                    for line in sequence_sql:
-                        cursor.execute(line)
+            self._reset_sequences(connection)
 
         if self.verbosity >= 1:
-            if self.fixture_object_count == self.loaded_object_count:
-                self.stdout.write(
-                    "Installed %d object(s) from %d fixture(s)"
-                    % (self.loaded_object_count, self.fixture_count)
-                )
-            else:
-                self.stdout.write(
-                    "Installed %d object(s) (of %d) from %d fixture(s)"
-                    % (self.loaded_object_count, self.fixture_object_count, self.fixture_count)
-                )
+            msg = "Installed %d object(s) {}from %d fixture(s)" % (
+                self.loaded_object_count, self.fixture_count)
+            of = ""
+            if self.fixture_object_count != self.loaded_object_count:
+                of = f"(of {self.fixture_object_count}) "
+            self.stdout.write(msg.format(of))
+
+    def _store_obj(self, obj):
```

Comment:
Same comment here as above comments. I'm guessing style wise though it's fine to leave out a docstring here as long as the name is informative enough (which I think all of them are).

Reply:
463566e467ebf21e9feffeb8c8d4c62a7d6e80c2